server/methods/groupsList.js

```diff
@@ -0,0 +1,37 @@
+Meteor.methods({
+	groupsList: function(nameFilter, limit, sort) {
+		if (!Meteor.userId()) {
+			throw new Meteor.Error('error-invalid-user', 'Invalid user', { method: 'groupsList' });
+		}
+
+		let options = {
+			fields: { name: 1 },
+			sort: { msgs: -1 }
+		};
+
+		//Verify the limit param is a number
+		if (_.isNumber(limit)) {
+			options.limit = limit;
+		}
+
+		//Verify there is a sort option and it's a string
+		if (_.trim(sort)) {
+			switch (sort) {
+				case 'name':
+					options.sort = { name: 1 };
+					break;
+				case 'msgs':
+					options.sort = { msgs: -1 };
+					break;
+			}
+		}
+
```

Comment:
Default sort is set above a few lines, but it was defaulted to messages and I have changed it to name.